src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.ts

```diff
@@ -90,6 +91,8 @@ export class ModelingAssessmentDashboardComponent implements OnInit, OnDestroy {
                 if (res.body!.type === ExerciseType.MODELING) {
                     this.modelingExercise = res.body as ModelingExercise;
                     this.getSubmissions(true);
+                    this.numberOfCorrectionrounds = this.modelingExercise.exerciseGroup ? this.modelingExercise!.exerciseGroup.exam!.numberOfCorrectionRoundsInExam! : 1;
+                    this.setPermissions();
                 } else {
                     // TODO: error message if this is not a modeling exercise
```

Comment:
Did you forget to implement this or is this intended? :D

Reply:
This statement was there before us, but yes we could remove it and implement a message

Reply:
Ah no I thought it was from you

Reply:
That was there before :D I'll check how this is handled in other exercise types, and then I'll either delete the else or add an error.